justfile

```diff
@@ -0,0 +1,37 @@
+set dotenv-load := false
+
+DOCKER_FILE := "-f docker-compose.yml"
+SERVICE_NAME := ""
```

Comment:
It doesn't seem like `SERVICE_NAME` is used - is there an intention for it in the future?

Reply:
Whoops, I think it's just left over. It's removed now :)